src/commands/__tests__/new.test.js

```diff
@@ -197,7 +197,7 @@ describe('project setup', () => {
 
   test('converts whitespaced project names to lowercased kebab-case', async () => {
     const name = {
-      raw:'archonauts string case', 
+      raw: 'archonauts string case',
```

Comment:
do we have lint in our project? if not, should we set it up?

Reply:
We have it, but we should change the current rules to catch this kind of issue.